backend/src/graphql/resolver/TransactionResolver.test.ts

```diff
@@ -1,12 +1,11 @@
-/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Why does this one stay here?

Reply:
![image](https://user-images.githubusercontent.com/1238238/235886234-619061b5-79ed-4df7-b794-d5b8178e9487.png)

I just investigated the connection stuff - for this test there is seed data variables which I did not adjust/research in and therefore the eslint ignore needs to be in place to pass the test